worker/dbaccessor/repl.go

```diff
@@ -68,7 +67,6 @@ func newREPL(pathToSocket string, dbGetter DBGetter, retryableError func(error)
 	r := &sqlREPL{
 		connListener:    l,
 		dbGetter:        dbGetter,
-		retryableError:  retryableError,
 		clock:           clock,
 		logger:          logger,
 		sessionCtx:      ctx,
```

Comment:
Apparently we're not supposed to do this...

Reply:
I know I know. I'll do a follow-up PR.